test/io-tests/test_io.py

```diff
@@ -684,6 +684,10 @@ def test_invalid_role_claim_key_notify_reload(defaultenv):
     with run(env=env) as postgrest:
         postgrest.session.post("/rpc/invalid_role_claim_key_reload")
 
+        # skips the first lines from stderr, the "Attempting to connect to database", "Connection successful", etc.
+        # this is a hack to avoid readline() from locking up the test
+        for _ in range(6):
+            postgrest.process.stderr.readline()
```

Comment:
This was the simplest choice for the test(there are [other](https://stackoverflow.com/questions/12419198/python-subprocess-readlines-hangs) [options](https://stackoverflow.com/questions/10756383/timeout-on-subprocess-readline-in-python), but they are complicated). The stderr output should be somewhat stable, so I think it's fine.